### ISSUE TITLE
Support better i18n/l10n than JBake

### ISSUE THREAD

**Issue by keilw** [razem-io/JBakeUI#1]
Running JBake natively through a batch or similar script, you end up with baked HTML files localized for the system's language. Thus, if you run it on a French or German OS locale sensitive items, especially date or time get rendered for this language (e.g. "Januar" instead of "January")

If JBakeUI allows to control this by calling JBake more directly, could you offer a "target language", please?

**Comment by razem-io** [comment]
JBakeUI actually runs JBake directly (using the jar file). Perhaps because it was compiled on a german system? I will look into this in a few hours.

**Comment by keilw** [comment]
No, if I download JBake and call it via a batch (see https://github.com/unitsofmeasurement/unitsofmeasurement-jbake) or similar ones at JavaMoney, the **runtime** locale seems to determine the HTML output. The JAR isn't touched or rebuilt, those scripts just use it. Guess applying something like "-Duser.country=CA -Duser.language=fr " in the batch script might help, but if JBakeUI calls the class from its JAR it would be cool, if it could do something like Locale.setDefault(myLocale) there based on a selection in the launcher.

**Comment by razem-io** [comment]
Ah I was a bit confused there! JBakeUI calls the class like any other batch script. So setting `java -jar -Duser.country=CA -Duser.language=fr` works perfectly. I am currently working on a settings dialog. Guess it will be finished by tomorrow. Then you will be able to set the country and the language.

**Comment by keilw** [comment]
OK, thanks, happy to test. Let us know, once it's available.

**Comment by razem-io** [comment]
Sorry for the slight delay. The feature is implemented now. Would be great if you could test it. You will find some empty list elements in the settings. This will be cleaned up for the 0.2 release.

**Comment by keilw** [comment]
Will do, there are some updates to either JSR 363, 354 or both I'm about to publish. So I'll try JBakeUI with those.
